### DISCUSSION TITLE
欧陆风云4

### DISCUSSION THREAD

**Discussion by luwenhegithub** [Gamuxorg/bbs#257] [投递和请求游戏]
游戏名称：欧陆风云4
游戏名称(英文)：Europa Universalis IV
下载地址 :（如果有的话）：

**Comment by wangling12** [comment]
#65 
这东西没有汉化,游戏引擎不支持中文
你要是对自己英文足够自信,我抽空可以上传一个😂😂

**Comment by luwenhegithub** [comment]
> #65
> 这东西没有汉化,游戏引擎不支持中文
> 你要是对自己英文足够自信,我抽空可以上传一个

不能汉化的确是个问题，我的英语也的确不咋地😂😂
